.yarn/versions/87656097.yml

```diff
@@ -0,0 +1,2 @@
+declined:
+  - "@nytimes/react-prosemirror"
```

Comment:
I think this is fine; this is a breaking change, but we haven't actually published this anywhere yet!